frontend/src/sass/core/_variables.scss

```diff
@@ -1,10 +1,12 @@
 $font-sans: 'Arial', 'helvetica';
 $font-sans-alt: 'Verdana';
 
-$transparent: rgba(0,0,0,0);
+$transparent: rgba(0, 0, 0, 0);
 $black: #212121;
 $dark-grey: #656565;
-$white: #FFFFFF;
+$light-grey: #cccccc;
+$white: #ffffff;
+$fs-yellow: #FEC33C;
```

Comment:
This color is not being used anywhere else, I recommend removing.

Reply:
@lauraGgit We are using fs-yellow in sprint-2, not sure about light-grey but we'll clean it up.

Reply:
@lpendem again, why isn't that just part of the existing color set on the /styleguide page?

Reply:
@lauraGgit this has been removed, and using existing colors already available.